src/open-websocket.js

```diff
@@ -17,5 +17,9 @@ export default url => {
     rws._ws.on('ping', pong)
   })
 
+  rws.addEventListener('close', () => {
+    rws.reconnet()
```

Comment:
Seem like a typo here cc @quantd2  `reconnet`

Reply:
`reconnect()` is one their API

Reply:
nvm, fixed